### PR TITLE
{bio}[foss/2020a] PIPITS v2.7 (+ ITSx, and version bumps for RDP-Classifier and VSEARCH)

### PR DESCRIPTION
**Pull request by bedroge**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by bedroge** [comment]
Test report by @bedroge
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
build.hpc.rug.nl - Linux centos linux 7.4.1708, Intel(R) Xeon(R) CPU E5-2660 0 @ 2.20GHz, Python 2.7.5
See https://gist.github.com/3e189708ea5263193ab5c0807e08d469 for a full test report.

**Comment by boegel** [comment]
`conda` is an easy escape here in the short term, but this is likely going to break in the not so distant future, since there's very little version restrictions on the dependencies.

I'm not sure how much of a mess this would be, but maybe trying a `PythonBundle` is worth a shot?

The documentation is rather poor, but the dependencies are listed in https://github.com/bioconda/bioconda-recipes/blob/master/recipes/pipits/meta.yaml.

**Comment by bedroge** [comment]
I went for the easy solution due to a lack of time ;-)
Maybe I'll have another look at this when I have some more time.

**Comment by bedroge** [comment]
Test report by @bedroge
**FAILED**
Build succeeded for 5 out of 7 (4 easyconfigs in this PR)
build.hpc.rug.nl - Linux centos linux 7.4.1708, x86_64, Intel(R) Xeon(R) CPU E5-2660 0 @ 2.20GHz, Python 2.7.5
See https://gist.github.com/4f405a78e765453c454bb0acdd77fc06 for a full test report.

**Comment by bedroge** [comment]
Test report by @bedroge
**SUCCESS**
Build succeeded for 7 out of 7 (4 easyconfigs in this PR)
build.hpc.rug.nl - Linux centos linux 7.4.1708, x86_64, Intel(R) Xeon(R) CPU E5-2660 0 @ 2.20GHz, Python 2.7.5
See https://gist.github.com/3db6eaec97221c5320c26b95cc422fa3 for a full test report.

**Comment by bedroge** [comment]
Test report by @bedroge
**SUCCESS**
Build succeeded for 8 out of 8 (5 easyconfigs in this PR)
build.hpc.rug.nl - Linux centos linux 7.4.1708, x86_64, Intel(R) Xeon(R) CPU E5-2660 0 @ 2.20GHz, Python 2.7.5
See https://gist.github.com/8c3020e4abff2e7ffa0b83953ba52565 for a full test report.

**Comment by bedroge** [comment]
@boegel I changed this from a Conda installation to an installation from source. It also includes version bumps for some existing easyconfigs (HMMER, RDP-Classifier, VSEARCH) and a new one (ITSx).

**Comment by bedroge** [comment]
Test report by @bedroge
**SUCCESS**
Build succeeded for 8 out of 8 (5 easyconfigs in this PR)
build.hpc.rug.nl - Linux centos linux 7.4.1708, x86_64, Intel(R) Xeon(R) CPU E5-2660 0 @ 2.20GHz, Python 2.7.5
See https://gist.github.com/73ef587eacd52f94ca353956dfb1d42f for a full test report.

File: easybuild/easyconfigs/i/ITSx/ITSx-1.1.2-GCCcore-9.3.0.eb [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+source_urls = ['https://microbiology.se/sw/']
+sources = ['%(name)s_%(version)s.tar.gz']
+checksums = ['45a9e99583189488e6cdfc37172732a330901ed713d92c8d5e7631a39fc4e72e']
+
+dependencies = [('Perl', '5.30.2')]
+
+fix_perl_shebang_for = ['ITSx']
+
+sanity_check_paths = {
+    'files': ['ITSx'],
```
Comment (boegel) [inline easybuild/easyconfigs/i/ITSx/ITSx-1.1.2-GCCcore-9.3.0.eb:21]: We like to have sanity check commands like "`ITSx --help`" or so, would that work here?

File: easybuild/easyconfigs/p/PIPITS/PIPITS-2.7-foss-2020a-Python-3.8.2.eb [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+    }),
+    (name, version, {
+        'source_tmpl': '%(version)s.tar.gz',
+        'source_urls': ['https://github.com/hsgweon/pipits/archive/'],
+        'checksums': ['ae2b3eca133a44c072bc0f0817af2affbcc1b2a64cc8904c326e4604cda3ecff'],
+    }),
+]
+
+sanity_check_paths = {
+    'files': ['bin/%s' % x for x in ['pipits_funits', 'pipits_process',
```
Comment (boegel) [inline easybuild/easyconfigs/p/PIPITS/PIPITS-2.7-foss-2020a-Python-3.8.2.eb:54]: Maybe add a sanity check command for one of these?

File: easybuild/easyconfigs/v/VSEARCH/VSEARCH-2.15.0-GCC-9.3.0.eb [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+    ('zlib', '1.2.11'),
+    ('bzip2', '1.0.8'),
+]
+
+preconfigopts = './autogen.sh &&'
+
+configopts = '--disable-pdfman '
+
+sanity_check_paths = {
+    'files': ['bin/vsearch'],
```
Comment (boegel) [inline easybuild/easyconfigs/v/VSEARCH/VSEARCH-2.15.0-GCC-9.3.0.eb:34]: Same here, add sanity check command?

**Review by boegel — CHANGES_REQUESTED** [review]

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 8 out of 8 (4 easyconfigs in total)
node3544.doduo.os - Linux RHEL 8.2, x86_64, AMD EPYC 7552 48-Core Processor (zen2), Python 3.6.8
See https://gist.github.com/acdd4cccd71a6898fa5c8b5d0cbf2783 for a full test report.

**Comment by boegel** [comment]
@boegelbot please test @ generoso

**Review by boegel — APPROVED** [review]

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 7 out of 7 (4 easyconfigs in total)
node3108.skitty.os - Linux centos linux 7.9.2009, x86_64, Intel(R) Xeon(R) Gold 6140 CPU @ 2.30GHz, Python 3.6.8
See https://gist.github.com/9851bf7bedf69d77782e43008297dca2 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 7 out of 7 (4 easyconfigs in total)
node2614.swalot.os - Linux centos linux 7.9.2009, x86_64, Intel(R) Xeon(R) CPU E5-2660 v3 @ 2.60GHz (haswell), Python 3.6.8
See https://gist.github.com/998f6c4ffb646a3fa785ec571d6af4eb for a full test report.

**Comment by boegel** [comment]
Going in, thanks @bedroge!

**Pull request merged**